### ISSUE TITLE
Experimental support error

### ISSUE THREAD

**Issue by CarlosV89** [TypeStrong/atom-typescript#1264]
Im getting `Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option to remove this warning`

This is my tslint.json:

```
{
  "compilerOptions": {
    "experimentalDecorators": true
  },
  "include": [
      "src/**/*"
  ],
  "exclude": [
      "node_modules",
      "**/*.spec.ts"
  ]
}
```

If I run tsc with --experimentalDecorators it compiles fine :/

**Comment by CarlosV89** [comment]
Compiler options are defined in tsconfig file, not tslint. This issue is not real.

**Comment by jdhines** [comment]
I'm using angular-cli, and it creates a tsconfig.json file with the `"experimentalDecorators": true` already, but I'm still getting this warning. How did you solve it?

**Comment by CarlosV89** [comment]
When I added the option to my tsconfig file it stopped happening.

**Comment by CarlosV89** [comment]
Make sure your task specifies the config file with --config and see if it still happens
